cache/sampleStore.js

```diff
@@ -101,7 +101,7 @@ function removeNullsAndStringifyArrays(obj, arrayFields) {
  * @returns {Object} cleaned up and ready to store in redis.
  */
 function cleanAspect(a) {
-  let retval = a.get();
+  let retval = a.get ? a.get() : a;
```

Comment:
maybe add a comment here or on line 100 suggesting that "a" can either be a sequelize "instance" object OR just a regular object?